netket/experimental/dynamics/_rk_solver.py

```diff
@@ -217,17 +216,17 @@ def general_time_step_adaptive(
         flags = set_flag_jax(is_at_max_dt, flags, SolverFlags.WARN_MAX_DT)
 
     # accept if error is within tolerances or we are already at the minimal step
-    accept_step = scaled_err < 1.0 or is_at_min_dt
+    accept_step = jnp.logical_or(scaled_err < 1.0, is_at_min_dt)
```

Comment:
I think this could be also written with `|` or whatever binary operator does OR...

Reply:
I find this easier to read, though.